test/relayHub/RelayHub.test.ts

```diff
@@ -1,1177 +1,507 @@
-        ERR_HUB_BAD_PARAMS
-      );
+      it('should require minimumEntryDepositValue > 0', async function () {
+        const promiseOfRevert = deployRelayHubContract([
+          undefined,
+          undefined,
+          '0',
+        ]);
 
-      const promiseOfRelayHub = deployRelayHubContract();
+        await expect(promiseOfRevert).to.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
 
-      await expect(promiseOfRelayHub).to.not.have.revertedWith(
-        ERR_HUB_BAD_PARAMS
-      );
-    });
+        const promiseOfRelayHub = deployRelayHubContract();
 
-    it('should require minimumUnstakeDelay > 0', async function() {
-      const promiseOfRevert = deployRelayHubContract([
-        undefined,
-        undefined,
-        undefined,
-        '0',
-      ]);
+        await expect(promiseOfRelayHub).to.not.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
+      });
 
-      await expect(promiseOfRevert).to.have.revertedWith(
-        ERR_HUB_BAD_PARAMS
-      );
+      it('should require minimumUnstakeDelay > 0', async function () {
+        const promiseOfRevert = deployRelayHubContract([
+          undefined,
+          undefined,
+          undefined,
+          '0',
+        ]);
 
-      const promiseOfRelayHub = deployRelayHubContract();
+        await expect(promiseOfRevert).to.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
 
-      await expect(promiseOfRelayHub).to.not.have.revertedWith(
-        ERR_HUB_BAD_PARAMS
-      );
-    });
+        const promiseOfRelayHub = deployRelayHubContract();
 
-    it('should require minimumStake > 0', async function() {
-      const promiseOfRevert = deployRelayHubContract([
-        undefined,
-        undefined,
-        undefined,
-        undefined,
-        '0',
-      ]);
+        await expect(promiseOfRelayHub).to.not.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
+      });
 
-      await expect(promiseOfRevert).to.have.revertedWith(
-        ERR_HUB_BAD_PARAMS
-      );
+      it('should require minimumStake > 0', async function () {
+        const promiseOfRevert = deployRelayHubContract([
+          undefined,
+          undefined,
+          undefined,
+          undefined,
+          '0',
+        ]);
+
+        await expect(promiseOfRevert).to.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
 
-      const promiseOfRelayHub = deployRelayHubContract();
+        const promiseOfRelayHub = deployRelayHubContract();
 
-      await expect(promiseOfRelayHub).to.not.have.revertedWith(
-        ERR_HUB_BAD_PARAMS
-      );
+        await expect(promiseOfRelayHub).to.not.have.revertedWith(
+          ERR_HUB_BAD_PARAMS
+        );
+      });
     });
-  });
 
-  describe('', function() {
-    beforeEach(async function() {
-      mockRelayHub = await deployRelayHubContract();
-    });
+    describe('', function () {
+      beforeEach(async function () {
+        mockRelayHub = await deployRelayHubContract();
```

Comment:
Why is this required here when the same instruction is present at line 47 in a more general beforeEach?

Reply:
Done